### DISCUSSION TITLE
Zero to Mastery first 25 hours.

### DISCUSSION THREAD

**Discussion by Keaton55** [mrdbourke/pytorch-deep-learning#661] [Q&A]
I wanted to continue this course after the youtube content so I signed up for Zero to Mastery for the rest of the course. Does anyone know anyway to mark the first 5 sections as watched so I don't have to watch those sections for the Certificate of completion?

**Comment by mrdbourke** [comment]
Hey @Keaton55 ,

Great question.

That would be good to be able to mark all of the first sections complete without having to watch them.

Let me check it out for you and let you know!

**Comment by mrdbourke** [comment]
Hey @Keaton55 ,

Quick update, at this stage we're not 100% sure how to mark each section as complete without actually going through each video.

I think this is by design of the platform (we use Teachable for ZTM) so people can't necessarily sign up and get the certificate without completing the materials.

**Comment by Keaton55** [comment]
Dang that is what I was worried about. Thanks for looking into it.
